### ISSUE TITLE
Liveness probe failed on Kubernetes hosted Concourse Worker

### ISSUE THREAD

**Issue by frayer** [concourse/concourse#3296]
## Bug Report

Using the `stable/concourse` Helm chart, I'm encountering an issue where the Kubernetes liveness probe of the Concourse Worker sporadically fails.  After the liveness probe fails, Kubernetes restarts the Concourse Worker Pod and logs an event on that Pod identical in nature to the issue described by Twistlock in this article:

https://www.twistlock.com/2018/12/04/advanced-runc-debugging-fun-profit/

They tracked their scenario to an issue in `runc`, and applied a fix to `runc 1.0-rc6`.

I'm recording this Issue for traceability because it's possible it could be fixed after updating the version of `runc` shipped with Concourse `4.2.x` as part of #3276.

## Steps to Reproduce

The issue is sporadic and I am unable to reproduce it on demand.  I'm able to observe when it happens through monitoring and then inspecting the Pod Restart Count of the Concourse Worker Pod(s).

## Expected Results

Kubernetes is able to exercise the liveness probe of the Concourse Worker as defined in the stable/concourse Helm chart.

## Actual Results

Kubernetes liveness probe on the Concourse Worker fails to execute, with an event like the following captured on the Pod after a restart:

```
  Type     Reason             Age   From                                    Message
  ----     ------             ----  ----                                    -------
  Warning  Unhealthy          5m    kubelet, <ip-address-removed>           Liveness probe failed: OCI runtime exec failed: exec failed: container_linux.go:348: starting container process caused "process_linux.go:90: adding pid 4692 to cgroups caused \"failed to write 4692 to cgroup.procs: write /sys/fs/cgroup/cpu,cpuacct/kubepods/burstable/pod0683367f-2e7d-11e9-8bf4-0242ac11000a/fe6dd88a32f48f481518faf51e31c685705a3647cc58fa386d0c9155d9de0130/cgroup.procs: invalid argument\"": unknown
```

## Additional Context

Potentially remediated by #3276

## Version Info

* Concourse version: 4.2.2
* Deployment type (BOSH/Docker/binary): Helm deployment of `stable/concourse --version 3.0.1`
* Infrastructure/IaaS: Kubernetes v1.11 cluster
* Browser (if applicable): N/A
* Did this used to work?  Have sporadically seen this issue for the past month

**Comment by vito** [comment]
@cirocosta Is this fixed by #2753 (and maybe more in the future with #3025)?

**Comment by cirocosta** [comment]
I *think* it *might* be.

It feels a lot related to how the Kubelet was trying to start the process inside the container to perform the healthcheck (which was using the `command` type before), but now that we moved to using `http`-based probes, that might not happen anymore.

Please let us know how experimenting with https://github.com/helm/charts/pull/11876 worked out for you, @frayer 😁
